### DISCUSSION TITLE
Opening Silk.NET window in MAUI?

### DISCUSSION THREAD

**Discussion by asheesh1996** [dotnet/Silk.NET#702] [Q&A] [answered]
Are there any examples of opening Silk.NET window in MAUI for Cross Platform development?

**Comment by Perksey** [comment, marked as answer]
It is in our plans for the 3.0 rewrite project, though this is still in the design phase at the moment: it's a long way off yet!

You can read more about our plans for the 3.0 rewrite here: #539 (MAUI is mentioned in the Software Development Plan)

**Reply by asheesh1996** [reply]
Hi, any update on this

**Reply by orosbogdan** [reply]
I would also be interested in an update? This could potentially be a very import feature for cross platform game development in .NET

**Reply by roeyskoe** [reply]
Only some ground work for 3.0 has started, see #864 and also [Proposal - 3.0 & 3.X Software Development Plan](https://github.com/dotnet/Silk.NET/blob/784a693501ec4ee14b87ac4b65a39229f185f870/documentation/proposals/Proposal%20-%203.0%20%26%203.X%20Software%20Development%20Plan.md).
Based on some recent Discord discussions, you could expect 3.0 to be released _maybe_ at the end of the year, or early 2023. With preview versions of course coming earlier.
